cli/upgrade.rs

```diff
@@ -65,7 +66,7 @@ pub async fn upgrade_command(
       Ok(ver) => {
         if !force && current_version == ver {
           println!("Version {} is already installed", &ver);
-          std::process::exit(1)
+          std::process::exit(0)
```

Comment:
I guess you can just return `Ok(())` since by default it will `exit(0)`

Reply:
true